src/main/java/seedu/address/model/client/Client.java

```diff
@@ -25,6 +27,7 @@
     private final Address address;
     private final Country country;
     private final Set<Tag> tags = new HashSet<>();
+    private final Set<Note> clientNotes = new HashSet<>();
```

Comment:
I would suggest that this is changed to a LinkedHashSet, as we will want the order of the notes to remain the same as the order they were added. HashSet does not guarantee order

I believe the country notes are also using LinkedHashSet

Also, I think it would be better if the clientNotes was initialized only if the client has notes, probably can add a TODO here to optimize this.

Reply:
Added todo, changed to LinkedHashSet